src/Utilities/ExcelImporterUtils.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace ExcelMapper.Utilities
+{
+    public static class ExcelImporterUtils
+    {
+#if NETSTANDARD2_0
+        public static List<ExcelClassMap> RegisterMapperClassesByNamespace(this ExcelImporter importer,
+            string namespacestr)
```

Comment:
I don't like this param name. Could you change it to `namespace`?

Reply:
namespace is reserved, I renamed it to nameSpace

Reply:
I updated this to be `namespaceString` - `nameSpace` is a bit weird IMO